tests/test_govuk_frontend.py

```diff
@@ -21,48 +21,17 @@ def question(self):
             }
         )
 
-    def test_govuk_input(self, question):
-        assert govuk_input(question) == {
-            "classes": "app-text-input--height-compatible",
-            "id": "input-title",
-            "name": "title",
-            "label": {
-                "classes": "govuk-label--l",
-                "isPageHeading": True,
-                "text": "What you want to call your requirements",
-            },
-            "hint": {
-                "html": '<div class="app-hint--text">\n'
-                'This will help you to refer to your requirements\n'
-                '</div>\n'
-                '100 characters maximum'
-            },
-        }
+    def test_govuk_input(self, question, snapshot):
+        assert govuk_input(question) == snapshot
 
-    def test_with_data(self, question):
+    def test_with_data(self, question, snapshot):
         data = {
             "title": "Find an individual specialist",
         }
 
-        assert govuk_input(question, data) == {
-            "classes": "app-text-input--height-compatible",
-            "id": "input-title",
-            "name": "title",
-            "label": {
-                "classes": "govuk-label--l",
-                "isPageHeading": True,
-                "text": "What you want to call your requirements",
-            },
-            "hint": {
-                "html": '<div class="app-hint--text">\n'
-                'This will help you to refer to your requirements\n'
-                '</div>\n'
-                '100 characters maximum'
-            },
-            "value": "Find an individual specialist",
-        }
+        assert govuk_input(question, data) == snapshot
 
-    def test_with_errors(self, question):
+    def test_with_errors(self, question, snapshot):
```

Comment:
Ah does `syrupy` match the test name exactly to the snapshot name then?

Reply:
Yes, it's super magical

Reply:
It all gets stored in the file [tests/__snapshots__/test_govuk_frontend.ambr](https://github.com/alphagov/digitalmarketplace-content-loader/pull/90/files#diff-4c0fcb4e0ec337b656861d12c91eda30) (ambr for amber, very cute).

I've seen other similar tools that use a pure Python file, but the advantage of this file format seems to be that it includes class information and deals with custom classes slightly better?

However the main reason I chose `syrupy` over `pytest-snapshot` or `snapshottest` is that it is being actively maintained, and it worked first time 😆